core/commands/files.go

```diff
@@ -440,6 +440,7 @@ being GC'ed.
 	},
 	Options: []cmds.Option{
 		cmds.BoolOption(filesParentsOptionName, "p", "Make parent directories as needed."),
+		cmds.BoolOption(forceOptionName, "f", "Force copy even if the node is not a valid UnixFS node."),
```

Comment:
This might be confused with the flush option, [here](https://github.com/ipfs/kubo/blob/master/core/commands/files.go#L76), so no shorthand.
```suggestion
		cmds.BoolOption(forceOptionName, "Force copy even if the node is not a valid UnixFS node."),
```

Reply:
Resolved